coldfront/api/allocation/tests/test_allocation_base.py

```diff
@@ -21,6 +23,19 @@ def setUp(self):
         """Set up test data."""
         super().setUp()
 
+        # Create default choices.
+        call_command('add_default_user_choices')
```

Comment:
The second command should already be called by a parent class (`TestAPIBase`). Could the first command be placed there too, and these two removed from here, without issue?